src/routes/endpoint.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::RwLock;
+
+use actix_web::{get, web, HttpResponse};
+
+use crate::{routes::error::Error, AppState};
+
+#[get("/api/v1/endpoint")]
+async fn get_endpoint(app_state: web::Data<RwLock<AppState>>) -> Result<HttpResponse, Error> {
+    let app_state = app_state.read()?;
+    let endpoint = match app_state.kilt_endpoint.as_str() {
+        "spiritnet" => "wss://kilt-rpc.dwellir.com",
+        _ => "wss://peregrine.kilt.io",
```

Comment:
That  is also super confusing.

```suggestion
        "spiritnet" => "wss://kilt-rpc.dwellir.com",
        "peregrine" => "wss://peregrine.kilt.io",
        other =>  other,
```

I could understand the above. But mapping everything to  peregrine? Even `wss://spiritnet.kilt.io`? That screams for  another 2h debugging session. 😓

Reply:
updated in: [a3552cb](https://github.com/KILTprotocol/opendid/pull/36/commits/a3552cb0ff9fe0ef3234c4df00b0a71df2b46fc6)